### ISSUE TITLE
Connection to paired device

### ISSUE THREAD

**Issue by neil-benn** [pybluez/pybluez#219]
I'm trying to connect to a device that we are already paired to; I'm making the call:

self.scontrol=BluetoothSocket(L2CAP)
        self.sinterrupt=BluetoothSocket(L2CAP)
        #self.pairinterrupt=BluetoothSocket(L2CAP)
        try:
            self.ccontrol,cinfo = self.scontrol.connect(('C8:FF:28:79:05:D4', 17))
            self.cinterrupt,cinfo = self.scontrol.connect(('C8:FF:28:79:05:D4', 19))
            connected = True
        except Exception as e:
            self.scontrol.shutdown(2)
            self.scontrol = None
            self.sinterrupt.shutdown(2)
            self.sinterrupt = None
            connected = False
            traceback.print_exc(e)

However when I make the call I get the error:

  File "server/btk_server.py", line 168, in listen
    self.ccontrol,cinfo = self.scontrol.connect(('C8:FF:28:79:05:D4', self.P_CTRL))
  File "<string>", line 5, in connect
BluetoothError: [Errno 111] Connection refused

  I can confirm that the adapter is there (you get a different error if it is not) but I just can't get the device to connect to the paired adapter (a windows PC).  Has anyone else resolved this at all?

**Comment by rgov** [comment]
Hi, you seem to have deleted the bug template that requested specific information about the operating system, Python version, etc. Could you please provide everything it asked for?

---

*PyBluez is not under active development but we are seeking new contributors
to investigate bugs and submit patches.*

## System

 - **Operating System:**
 - **Hardware:** (e.g., Raspberry Pi, external Bluetooth adaptor, etc.)
 - **PyBluez Version:**


## Issue

Please provide information necessary to reproduce the issue, including sample
code.

If you have long logs, please post them on https://gist.github.com and link to
the Gist here.

**Comment by neil-benn** [comment]
Apologies:

OS - "Raspbian GNU/Linux 9 (stretch)"
Hardware - Raspberry Pi Zero W
PyBluez - version 0.22 (also tried with 0.17)
Python version - 2.7.13

  The full code is below; note that the local device mac address is hardcoded as is the windows pc adapter address.  I have to change the file ending to get github to accept it:

[btk_server.py.txt](https://github.com/pybluez/pybluez/files/2435992/btk_server.py.txt)

  I'm happy to share the code publicy when it is done because I picked a lot of it up off the Bluetooth keyboard example and I'm happy to contribute to the project in general in the future once I've got this up and off the ground.  Note that there are reputation points up for grabs on stackoverflow on this as well!  It's 12:30 am here so I'm off to bed!
